mobile_verifier/src/heartbeats.rs

```diff
@@ -19,26 +18,75 @@ use sqlx::{Postgres, Transaction};
 use std::{ops::Range, pin::pin, sync::Arc, time};
 use tokio::sync::mpsc::Receiver;
 
-#[derive(Debug, Clone, PartialEq, Eq, Hash, sqlx::FromRow)]
-pub struct HeartbeatKey {
-    hotspot_key: PublicKeyBinary,
-    cbsd_id: String,
+/// Minimum number of heartbeats required to give a reward to the hotspot.
+const MINIMUM_CELL_HEARTBEAT_COUNT: i64 = 12;
+const MINIMUM_WIFI_HEARTBEAT_COUNT: i64 = 12;
+
+#[derive(Clone)]
+enum HBType {
```

Comment:
HBType was added so as to be able to explicitly specify/determine the cell type of a heartbeat.  Prior to adding this, at the point of validating a HB, we had to implicitly assume a WIFI cell type based on the lack of `cbsd_id`.  NOTE: a label can only be determined after we have determined the cell type...I prefer to be explicit here rather than implicit